docs/how-to/c-functions.rst

```diff
@@ -63,34 +63,35 @@ Once you have opened the relevant header file in a text editor, you need to sear
 
     int puts(const char *);
 
-This means that ``puts`` returns an ``int`` and takes a single argument of type ``const char *`` (a C string). This translates to the following Python ``ctypes`` code:
+This means that ``puts`` returns an ``int`` and takes a single argument of type ``const char *`` (a pointer to one or more characters, i. e. a C string). This translates to the following Python ``ctypes`` code:
```

Comment:
Typo: space between in `i. e.`

Reply:
Yeah - in German you *have* to put a space there ([we literally have a standard for this](https://de.wiktionary.org/wiki/z._B.) - [DIN 5008](https://en.wikipedia.org/wiki/DIN_5008) 😛), so I tend to do it in English too by habit.

Reply:
Huh.. TIL. Sadly, English has no such central authority, so we have to fall back on documents like the [Chicago manual of style](https://www.amazon.com/gp/product/0226104206/ref=as_li_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=0226104206&linkCode=as2&tag=dwt0d9-20&linkId=798f352a8de37926fe606c37b89b4611), [AP Style Guide](https://www.amazon.com/gp/product/0465062946/ref=as_li_tl?ie=UTF8&camp=1789&creative=9325&creativeASIN=0465062946&linkCode=as2&tag=dwt0d9-20&linkId=0e9d37192ce4793e003e21a7d66f4167), all of which are proprietary documents you can't link to.